sdk/cosmos/azure-cosmos-benchmark/ctl/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:18.04
```

Comment:
Can the docker publishing and consuming latest also be automated?

Reply:
Is that the test CI referred in IM?

Reply:
We could use "FROM ubuntu:latest" which will bring ubuntu:18.04 which is current latest LTS version.
I still prefer mentioning version , so when we change it manually and see some different result we can relate to the version change.